cpp/include/raft/neighbors/ivf_pq.cuh

```diff
@@ -196,6 +200,17 @@ inline void search(const handle_t& handle,
     handle, params, index, queries, n_queries, k, neighbors, distances, mr);
 }
 
+template <typename IdxT>
+void save(const handle_t& handle, const std::string& filename, const index<IdxT>& index_)
```

Comment:
We should add doxygen here and make a strong note that these are experimental (here and ivf_flat) and so they are subject to change.